src/modinv64_impl.h

```diff
@@ -145,72 +145,72 @@ typedef struct {
     int64_t u, v, q, r;
 } secp256k1_modinv64_trans2x2;
 
-/* Compute the transition matrix and eta for 59 divsteps (where zeta=-(delta+1/2)).
+/* Compute the transition matrix and theta for 59 divsteps (where theta=delta-1/2)
  * Note that the transformation matrix is scaled by 2^62 and not 2^59.
```

Comment:
Is it worth elaborating on some of these parameter choices?

```suggestion
/* Compute the transition matrix and theta for 59 divsteps (where theta=delta-1/2)
 * We use 59 instead of 62 because we need 590 total divsteps.
 * However, we scale the transformation matrix by 2^62 and not 2^59 to
 * use the precomputed inverse of modulus mod 2^62.
 */
```

Reply:
How's this?

```
/* Compute the transition matrix and theta for 59 divsteps (where theta=delta-1/2)
 * Although only 59 divsteps are performed, the resulting transformation matrix
 * is scaled by 2^62 to allow reuse of _update_de_62 and _update_fg_62 between
 * _modinv64 and _modinv64_var.
```